From d4d1be386f1c49eae348a5008e2a19073678a097 Mon Sep 17 00:00:00 2001
From: Michael Biebl <biebl@debian.org>
Date: Thu, 16 Apr 2009 18:27:07 +0200
Subject: [PATCH 02/53] lib: fix make distcheck for lksctp-tools

lksctp-tools-1.0.10 currently fails "make distcheck".
Problem is, that src/lib/Version.map is not correctly referenced in
src/lib/Makefile.am leading to problems when builddir != srcdir.

From: Michael Biebl <biebl@debian.org>
Signed-off-by: Vlad Yasevich <vladislav.yasevich@hp.com>
---
 src/lib/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/lib/Makefile.am b/src/lib/Makefile.am
index 93be8a1..f14f444 100644
--- a/src/lib/Makefile.am
+++ b/src/lib/Makefile.am
@@ -9,4 +9,4 @@ INCLUDES = -I$(top_srcdir)/src/include
 
 lib_LTLIBRARIES = libsctp.la
 libsctp_la_SOURCES = bindx.c connectx.c peeloff.c opt_info.c addrs.c sendmsg.c recvmsg.c Versions.map
-libsctp_la_LDFLAGS = -version-info 1:10:0 -Wl,--version-script=Versions.map
+libsctp_la_LDFLAGS = -version-info 1:10:0 -Wl,--version-script=$(srcdir)/Versions.map
-- 
1.7.11.7

